app/graphql/resolvers/concerns/collection.rb

```diff
@@ -7,13 +7,17 @@ module Collection
       extend ActiveSupport::Concern
 
       include Sorting
+      include TagFiltering
 
       included do
         type ["Types::#{self::MODEL_CLASS}".safe_constantize], null: false
         argument :search, String, 'Search query', required: false
         argument :limit, Integer, 'Pagination limit', required: false
         argument :offset, Integer, 'Pagination offset', required: false
         argument :sort_by, [String], 'Sort results', required: false
+        if self::MODEL_CLASS.column_names.include?('tags')
```

Comment:
If the `tags_supported?` method is made a static one (on `TagFiltering` module) the check can be kept on one place.
```
if TagFiltering.tags_supported?(self::MODEL_CLASS)
  ...
```

```
module TagFiltering
  def self.tags_supported?(resource)
    resource.column_names.include?('tags')
  end

...
```